### ISSUE TITLE
ToB start fade out does not get cancelled any more 

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#1763]
I swear this used to work, it'd be obvious when release testing. Looking back at appimages, it was a problem already in 0.9.0, so I suspect SDL2 is somehow involved. I don't have SDL1 any more to try, but opengl doesn't matter either. Not that any of it should, since we just launch a long fade in hack in CharGenEnd and then core is supposed to cancel it. Opening and closing windows doesn't help. But, the round immediate exit window does come up with the fade cleared. After closing it, it is restored, so maybe just a modal thing.

TLDR; can you start ToB normally without it taking ages to fade in properly?

**Comment by bradallred** [comment]
The fade hack is canceled in BG2 by a scripted fade. I don't think ToB was ever an intended target of the fade in (I never checked the original).

**Comment by lynxlynxlynx** [comment]
works fine without 🤷🏾
